src/features/unified-share-modal/ContactsField.js

```diff
@@ -28,6 +28,7 @@ type Props = {
     onContactAdd: Function,
     onContactRemove: Function,
     onInput?: Function,
+    onPillCreate?: Function,
```

Comment:
For new additions please add full flow type (stuff) => void

Reply:
done